### PR TITLE
Add initial yes/no page for 10-10CG POA

### PR DESCRIPTION
**Pull request by LinnJS**
## Description
Add initial yes/no/no question POA page. This question is to determine if the user would like to sign as representative and upload document or not to upload document or not to sign as representative. Make sure the content is 1:1 with designs attached in this ticket below.

[Ticket](https://app.zenhub.com/workspaces/vsa---caregiver-5fff0cfd1462b6000e320fc7/issues/department-of-veterans-affairs/va.gov-team/21859)

## Designs
image

image

## Acceptance criteria
- [x] We have the explanation intro content at the top of the page
- [x] We are collecting the yes/no/no data
- [x] We are showing the additional info at the bottom of the page
- [x] We are feature flipping this page so it does not go to prod unfinished or untested

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/caregivers/config/chapters/signAsRepresentative/signAsRepresentativeYesNo.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+// import fullSchema from 'vets-json-schema/dist/10-10CG-schema.json';
+import { representativeFields } from 'applications/caregivers/definitions/constants';
+import AdditionalInfo from '@department-of-veterans-affairs/component-library/AdditionalInfo';
+
+// const { representative } = fullSchema.properties;
```
Comment (LinnJS) [inline src/applications/caregivers/config/chapters/signAsRepresentative/signAsRepresentativeYesNo.js:6]: Leaving this in here as a reminder to integrate schema but will be merging as functional under a feature flip for testing

File: src/applications/caregivers/config/form.js [hunk 2]
```diff
@@ -120,6 +123,18 @@ const formConfig = {
         },
       },
     },
+    signAsRepresentativeChapter: {
+      title: 'Representative documentation',
+      pages: {
+        signAsRepresentative: {
+          depends: formData => formData['view:canUpload1010cgPOA'],
```
Comment (LinnJS) [inline src/applications/caregivers/config/form.js:130]: Feature flip reference to toggle the page based off flip setting

File: src/applications/caregivers/config/chapters/signAsRepresentative/signAsRepresentativeYesNo.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+// const veteranProps = veteran.properties;
+// const { address, phone } = fullSchema.definitions;
+
```
Comment (LinnJS) [inline src/applications/caregivers/config/chapters/signAsRepresentative/signAsRepresentativeYesNo.js:9]: The page its self

File: src/applications/caregivers/config/chapters/signAsRepresentative/signAsRepresentativeYesNo.js [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+const RepresentativeAdditionalInfo = () => {
+  return (
+    <div className="vads-u-margin-top--1">
+      <AdditionalInfo triggerText="What documents can I submit to show legal status as a representative?">
+        <ul>
+          <p>
+            Documentation to show your legal status as a representative could
+            include:
+          </p>
```
Comment (Mottie) [inline src/applications/caregivers/config/chapters/signAsRepresentative/signAsRepresentativeYesNo.js:44]: A `<p>` shouldn't be a child of a `<ul>`

**Review by Mottie — APPROVED** [review]
ghe-emoji

File: src/applications/caregivers/config/chapters/signAsRepresentative/signAsRepresentativeYesNo.js (hunk 4 above)
Comment (LinnJS) [reply src/applications/caregivers/config/chapters/signAsRepresentative/signAsRepresentativeYesNo.js]: Yeah I was thinking about that I moved it out due to a indent but I think you are right just need to remove that indent

Comment (LinnJS) [reply src/applications/caregivers/config/chapters/signAsRepresentative/signAsRepresentativeYesNo.js:48]: 100% good catch!

**Review by craigwheeler — APPROVED** [review]

**Pull request merged**